gymnasium/experimental/wrappers/lambda_reward.py

```diff
@@ -93,7 +93,7 @@ def __init__(
         super().__init__(env, lambda x: np.clip(x, a_min=min_reward, a_max=max_reward))
 
 
-class NormalizeRewardV0(gym.Wrapper):
+class NormalizeRewardV1(gym.Wrapper):
     r"""This wrapper will normalize immediate rewards s.t. their exponential moving average has a fixed variance.
```

Comment:
Could you add to the documentation, the information from this PR (https://github.com/openai/gym/pull/3152) to explain the decision to increase the version number

Reply:
I wasn't sure how to properly write it

Reply:
```
Note: 
    In v0.27, NormalizeReward was updated as the forward discounted reward estimate was incorrect computed in Gym v0.25+. For more detail, read [#3154](https://github.com/openai/gym/pull/3152)
```

Reply:
Thank you! Fixed